vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -54,11 +54,14 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.8.0",
     # azuredisk-csi:v1.28 is only for AKS 1.27+, v1.26 is for other AKS versions
     "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.5",
-    "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.28.0-windows-hp",
+    "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.6",
+    "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.28.1-windows-hp",
```

Comment:
Is v1.28.1-windows-hp being used?

Reply:
yes, `v1.28.1-windows-hp` is already used in aks-rp